.goreleaser.yml

```diff
@@ -107,8 +112,7 @@ dockers:
       - nri-prometheus
     image_templates:
       - 'newrelic/nri-prometheus:{{ .Major }}.{{ .Minor }}'
-    # auto mean that if version has rc-1 or similar it won't be pushed
```

Comment:
rc-1?

Reply:
Nonsense from the goreleaser docs. Apparently it won't push the image if it detects the tag has some `rc` suffixes or something like that. Since we are not using it and I didn't want that to bite us in the ass, I stripped it.